src/components/Theming.ts

```diff
@@ -21,6 +41,9 @@ const themes = {
       filterTagsText: colors.mineShaft,
       ...colors,
     },
+    toggleTheme: () => {
+      // noop
+    },
```

Comment:
I defined it here but it's set in `src/components/Layout.tsx` where we use `ThemeProvider`

Reply:
Ah good to know!